### PR TITLE
Added scanlines to render.createFont

### PR DESCRIPTION
**Pull request by spankyyy**
parity with the starfall font editor

### CODE REVIEW

File: lua/starfall/libs_cl/render.lua [hunk 1]
```diff
@@ -1445,7 +1445,7 @@ end
 -- \- Lucida Console
 -- \- Times New Roman
 
-function render_library.createFont(font, size, weight, antialias, additive, shadow, outline, blur, extended)
+function render_library.createFont(font, size, weight, antialias, additive, shadow, outline, blur, extended, scanlines)
```
Comment (thegrb93) [inline lua/starfall/libs_cl/render.lua:1448]: Update the doc above this too

File: lua/starfall/libs_cl/render.lua [hunk 2]
```diff
@@ -1471,13 +1471,13 @@ function render_library.createFont(font, size, weight, antialias, additive, shad
 		surface.CreateFont(name, { size = size, weight = weight,
 			antialias = antialias, additive = additive, font = font,
 			shadow = shadow, outline = outline, blur = blur,
-			extended = extended })
+			scanlines = scalines, extended = extended })
```
Comment (thegrb93) [inline lua/starfall/libs_cl/render.lua:1474]: typo here

**Comment by Derpius** [comment]
FYI scanlines has certain limitations which don't look like they can be localised to individual chips/players, which may be why this wasn't implemented in the first place
image

**Comment by thegrb93** [comment]
Superceded with https://github.com/thegrb93/StarfallEx/pull/1263

**Pull request closed without merging**